webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlQuery.scala

```diff
@@ -573,19 +573,32 @@ case class OrderCriterion(queryVariable: QueryVariable, isAscending: Boolean) ex
     }
 }
 
+/**
+ * Represents a FROM clause.
+ *
+ * @param namedGraph the named graph to be used in the query.
+ */
+case class FromClause(namedGraph: IriRef) extends SparqlGenerator {
+    override def toSparql: String = s"FROM ${namedGraph.toSparql}\n"
```

Comment:
we have a named graph here, shouldn't we use `FROM NAMED`? Or are you using `FROM` because the namedGraph is anyway the default graph? In that case, please rename the `nameGraph` to `defaultGraph`.

Reply:
Sorry, this is confusing. For me, any graph in the triplestore that has a name is a "named graph". The pseudo-graph `<http://www.ontotext.com/explicit>` is a "named graph" in that sense:

http://graphdb.ontotext.com/documentation/standard/query-behaviour.html#how-to-query-explicit-and-implicit-statements

I've clarified this in a7f3a1cc.